packages/api/tests/endpoints/test-elasticsearch.js

```diff
@@ -267,6 +267,12 @@ test.serial('Reindex request returns 400 with the expected message when source i
 test.serial('Reindex request returns 400 with the expected message when source index matches the default destination index.', async (t) => {
   const date = new Date();
   const defaultIndexName = `cumulus-${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`;
+
+  try {
```

Comment:
Yeah I had also fixed this test thusly:

https://github.com/nasa/cumulus/pull/2251/commits/4e699ca8c7ab1fe61f9bda7702920e665a2e9d73

Reply:
Hmm, does `body: { mappings }` not matter? 🤔

Reply:
I don't think it does because the test isn't actually indexing any data, it's just testing to prove that the endpoint throws an error if the destination index matches the default. So we all care about is that the default index name exists

Reply:
Ah, yeah, 👍🏻  forest for the trees.      3415eba6d